fastlane-plugin-wpmreleasetoolkit.gemspec

```
@@ -28,6 +28,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'diffy', '~> 3.3'
   spec.add_dependency 'nokogiri', '~> 1.11' # Needed for AndroidLocalizeHelper
   spec.add_dependency 'octokit', '~> 4.18'
+  spec.add_dependency 'buildkit', '~> 1.4.5'
+
```

Have you considered relaxing to:
 
```suggestion
  spec.add_dependency 'buildkit', '~> 1.4'

```

---

Also, is the empty line intentional to differentiate the block of dependencies above from the one below?

In case it isn't:

```suggestion
  spec.add_dependency 'buildkit', '~> 1.4.5'
```

---

Good call on these, thanks!